### PR TITLE
Add more logging to vmexec server

### PR DESCRIPTION
**Pull request by bduffany**
**Related issues**: N/A

### CODE REVIEW

File: enterprise/server/vmexec/vmexec.go [hunk 1]
```diff
@@ -322,11 +361,45 @@ func newCommand(start *vmxpb.ExecRequest) (*command, error) {
...
 
+func (c *command) logDebugInfo(err error) {
+	log.Warningf("Command terminated abnormally: %s", err)
+	// Check integrity of the executable
+	if len(c.cmd.Args) > 0 {
+		executable := c.cmd.Args[0]
+
+		realpath, err := exec.LookPath(executable)
+		if err != nil {
+			log.Warningf("Failed to look up executable in $PATH: %s", err)
```
Comment (maggie-lou) [inline enterprise/server/vmexec/vmexec.go:372]: Should we exit early here?

File: enterprise/server/vmexec/vmexec.go [hunk 2]
```diff
@@ -322,11 +361,45 @@ func newCommand(start *vmxpb.ExecRequest) (*command, error) {
 	}, nil
 }
 
+func (c *command) logDebugInfo(err error) {
```
Comment (maggie-lou) [inline enterprise/server/vmexec/vmexec.go:364]: Should we do some error filtering here? There can be lots of uninteresting reasons a command fails

File: enterprise/server/vmexec/vmexec.go (hunk 1 above)
Comment (bduffany) [reply enterprise/server/vmexec/vmexec.go]: Done

File: enterprise/server/vmexec/vmexec.go (hunk 2 above)
Comment (bduffany) [reply enterprise/server/vmexec/vmexec.go]: Done, filtering to "exec format error" for now

**Review by jdhollen — APPROVED** [review]

**Pull request merged**